content/3_Silver/Binary_Search.mdx

```diff
@@ -368,44 +361,40 @@ def lastTrue(f, lo, hi):
 		mid = (lo+hi)//2 # find the middle of the current range
 		if f(mid):
 			# if mid works, then all numbers smaller than mid also work
-			# so we only care about the part after mid
 			res = mid # update the answer
 			lo = mid+1 # cut the part before mid
 			# notice that we already handled mid itself so we cut it as well
 		else:
 			# if mid does not work, greater values would not work too
-			# so we don't care about them
 			hi = mid-1 # cut mid and after
 	# now res is the answer
 	# if res is l-1 that means no value in the range satisfies the condition
 	return res
-print(lastTrue(lambda x:True, 2, 10)) # 10
+print(lastTrue(2, 10,lambda x:True)) # 10
 # all numbers satisfy the condition
-print(lastTrue(lambda x:x*x <= 30, 2, 10)) # 5
-print(lastTrue(lambda x:False, 2, 10)) # 1
+print(lastTrue(2, 10,lambda x:x*x <= 30)) # 5
+print(lastTrue(2, 10,lambda x:False)) # 1
```

Comment:
Add a space after `10,` (also please fix the spacing in the lambdas if you don't mind)